test/tools/unified-spec-runner/schema.ts

```diff
@@ -199,6 +200,7 @@ export type EntityDescription =
   | { database: DatabaseEntity }
   | { collection: CollectionEntity }
   | { bucket: BucketEntity }
+  | { thread: UnifiedThread }
```

Comment:
```suggestion
  | { thread: Pick<UnifiedThread, 'id'> }
```

Just because the current types imply that `thread:` maps to an instance of UnifiedThread, so `Pick` or making a `ThreadEntity` interface (despite the redundancy, I know) will represent the reality better.

Reply:
yes, good catch, this was just leftover code from before I named things appropriately and realized that the actual entity only has an id